src/components/Table/Table.stories.tsx

```diff
@@ -178,3 +192,60 @@ export const fixed = (): React.ReactElement => (
     </tbody>
   </Table>
 )
+
+export const scrollable = (): React.ReactElement => (
+  <Table scrollable={true} caption="This is a scrollable table">
+    {testContent}
```

Comment:
In the USWDS example, Scrollable table uses different data than the other tables, presumably because of its intended purpose:

> A scrollable table is ideal for dense, tabular data.

Can you create a set of test data for the scrollable table by copying what USWDS uses in their example to use in this story?